ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -310,6 +304,58 @@ TFLiteCore::getOutputTensorDim (GstTensorsInfo * info)
   return 0;
 }
 
+/**
+ * @brief set the Dimension for Input Tensor.
+ * @param info Structure for input tensor info.
+ * @return 0 if OK. non-zero if error.
+ * @note rank can be changed dependent on the model
+ */
+int
+TFLiteCore::setInputTensorDim (const GstTensorsInfo * info)
+{
+  TfLiteStatus status;
+  const std::vector<int> &input_idx_list = interpreter->inputs ();
+
+  /** Cannot change the number of inputs */
+  if (info->num_tensors != input_idx_list.size ())
+    return -EINVAL;
+
+  for (int tensor_idx = 0; tensor_idx < info->num_tensors; ++tensor_idx) {
+    std::vector<int> dims;
```

Comment:
Do ```std::vector<int> dims(NNS_TENSOR_RANK_LIMIT);``` and remove Line 330

Reply:
Removed reserve and directly allocated memory.